src/i18n.ts

```diff
@@ -181,27 +185,23 @@ should either enable sessions or use `ctx.i18n.useLocale()` instead.",
       useLocale(negotiatedLocale);
     }
 
-    // Also exports ctx object properties for accessing them directly from
-    // the translation source files.
-    function translateWrapper(
-      key: string,
-      translationContext?: TranslationContext,
-    ): string {
-      return translate(key, {
-        ...globalTranslationContext?.(ctx),
-        ...translationContext,
-      });
-    }
-
     ctx.i18n = {
       fluent,
       renegotiateLocale: negotiateLocale,
       useLocale,
       getLocale: getNegotiatedLocale,
       setLocale,
     };
-    ctx.t = translateWrapper;
-    ctx.translate = translateWrapper;
+    ctx.t = <K extends string>(
+      key: string,
+      translationContext?: TranslationVariables<K>,
+    ): string => {
+      return translate(key, {
+        ...globalTranslationContext?.(ctx),
+        ...translationContext,
+      });
+    };
+    ctx.translate = ctx.t;
```

Comment:
I would like use `ctx.translate` to define function and `ctx.t` as an alias. It will be more readable that way if we go like this.

```Typescript
ctx.translate = <K extends string>(
  key: string,
  translationContext?: TranslationVariables<K>,
): string => {
  return translate(key, {
    ...globalTranslationContext?.(ctx),
    ...translationContext,
  });
};
ctx.t = ctx.translate;
```

This way developers can easily understand what the function is about.